app/views/pages/privacy.html.haml

```diff
@@ -0,0 +1,24 @@
+- content_for :page_title do
+  Privacy Policy
+
+%h2.ui.header Privacy Policy
+
+%p
+  Privacy takes the front stage in transientBug. That being said, don't store sensitive information.
+  All user data is stored on transientBug servers, and the official web extensions only communicate with the transientBug API.
+
+%p
+  The following data is sent by the official web extension when its page action is clicked: The active tabs Title and URL. That's it.
```

Comment:
you should add something about what happens when an account is deleted

Reply:
goooooooood point. except you can't delete your account yet.

well no you can, if you email me I guess

Reply:
commenting on commits is a bad idea, isn't it